source/wp-content/themes/wporg-developer-2023/parts/search-wide.html

```diff
@@ -0,0 +1,7 @@
+<!-- wp:group {"style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}},"className":"alignfull","layout":{"type":"default"}} -->
+<div class="wp-block-group alignfull" style="padding-right:var(--wp--preset--spacing--edge-space);padding-left:var(--wp--preset--spacing--edge-space)">
```

Comment:
I saw these changes on the other a11y branch fix: https://github.com/WordPress/wporg-developer/pull/388/files#diff-64e65ccda0f0264641251aef7432f56f0916d0f922c6312b3387553ab6be2d25R2

Reply:
Ah both these branches need rebasing after the layout branch was merged, thanks